src/components/CMessageRow.vue

```diff
@@ -1,5 +1,8 @@
 <template>
-  <tr class="border-b border-gray-200 hover:bg-blue-100" @click="onToggle(messageId)">
+  <tr
```

Comment:
Charles also asked for a caret and i think it's a good idea.
Maybe set message_id as first column, add a > char at the beginning, \/ when it's open (find nice svg or utf8 characters) to do that.